src/sentry/replays/query.py

```diff
@@ -703,6 +734,11 @@ def _empty_uuids_lambda():
         parameters=[Column("count_info_events")],
         alias="count_infos",
     ),
+    "viewed_by_ids": Function(
+        "groupUniqArray",
```

Comment:
We need to remove the `0` ids.  We can ues `groupUniqArrayIf` which accepts a second argument which can be `x -> x > 0`.  Or something similar.  There's prior art in this file with `anyIf`.